app/views/Data/Workspace/WorkTable/index.tsx

```diff
@@ -183,8 +232,26 @@ export default function WorkTable(props: Props) {
                     </Menu.Target>
 
                     <Menu.Dropdown>
-                        <Menu.Item icon={<MdGrid3X3 />}>Number</Menu.Item>
-                        <Menu.Item icon={<MdHdrAuto />}>String</Menu.Item>
+                        <Menu.Item
+                            name="number"
+                            icon={<MdGrid3X3 />}
+                            onClick={(e) => handleMenuClick(
+                                e.currentTarget.name,
+                                col?.key,
+                            )}
+                        >
+                            Number
+                        </Menu.Item>
+                        <Menu.Item
+                            name="string"
+                            icon={<MdHdrAuto />}
+                            onClick={(e) => handleMenuClick(
+                                e.currentTarget.name,
+                                col?.key,
+                            )}
```

Comment:
This function is duplicated. Same as the one for number. Please make it a single function.